src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -49,6 +49,25 @@ namespace drt {
 
 using utl::ThreadException;
 
+// TODO there should be a better way to get this info by getting the master
```

Comment:
It seems this comment is outdated now, since you are already using odb to get the info.